front/src/modules/rollingStock/components/RollingStockEditor/CurveParamSelectors.tsx

```diff
@@ -244,7 +244,10 @@ const CurveParamSelectors = ({
     );
   };
 
-  const isWarning = Object.values(extraColumnData.data).includes('');
+  const isWarning = Object.values(extraColumnData.data).reduce(
```

Comment:
warningsNb or warningsCount ?
```suggestion
  const warningsNb = Object.values(extraColumnData.data).reduce(
```

Reply:
Fixed, ty for the comments !